lib/core/include/rcMisc.h

```diff
@@ -349,6 +349,22 @@ int get_canonical_name(const char* _hostname, char* _buf, size_t _len);
 
 int load_in_addr_from_hostname(const char* _hostname, struct in_addr* _out);
 
+/// Determines whether the given buffer could contain sensitive data.
+///
+/// The contents of the buffer is considered sensitive if it contains any of the following
+/// byte sequences:
+/// - <authPlugReqInp_PI>
+///
+/// \param[in] _buffer      The buffer.
+/// \param[in] _buffer_size The size of the buffer.
+///
+/// \return A boolean value.
+/// \retval true  If the buffer might contain sensitive data.
+/// \retval false Otherwise.
+///
+/// \since 4.2.12
+bool might_contain_sensitive_data(const char* _buffer, size_t _buffer_size);
```

Comment:
might -> may ?

not convinced, just a suggestion...

Reply:
updated